### ISSUE TITLE
[use-in-view] Image component, last image of page not loading

### ISSUE THREAD

**Issue by heyimbrucelee** [frontity/frontity#241]
**_Bug report_**

- [x] I've searched for existing issues
- [x] I've checked documentation: https://docs.frontity.org
- [ ] I've checked community forum: https://community.frontity.org

**Describe the expected behavior**

Image component should lazy load images

**Describe the observed behavior**

I added a Image component in the Loading component.
Loading is really fast so the Image is quickly added & removed and sometimes isn't displayed on first load w/e issue is not about this image.

Its sometimes lead to this error :

```
use-in-view.ts:formatted:22 Uncaught TypeError: Cannot read property '0' of undefined
    at eval (use-in-view.ts:formatted:22)
    at Array.forEach (<anonymous>)
    at IntersectionObserver.eval (use-in-view.ts:formatted:19)
```

and last Image of page doesnt lazy load.

In hook use-in-view :

```
 return () => {
      observer.unobserve(ref.current);
    
      setFunctions.splice(
        setFunctions.findIndex(set => set[1].current === ref.current),
        1
      );
    };
}
```

setFunctions.findIndex(set => set[1].current === ref.current) can sometimes return -1
and delete last element of array --> issue loading last image

Solved it ensuring the function was found, not sure if it is the good way.
```
const idx = setFunctions.findIndex(set => set[1].current === ref.current);
idx > 0 && setFunctions.splice(idx,1);
```

**Comment by luisherranz** [comment]
The fix sounds good to me. Good work @heyimbrucelee :)

Could you please submit a pull request with the fix?
These are the instructions on how to contribute: https://docs.frontity.org/contributing/code-contributions

**Comment by michalczaplinski** [comment]
This should have been fixed in #251 and will be available in the next release. Let me know if you will still run into this problem @heyimbrucelee 🙂
